vsintegration/src/FSharp.Editor/Structure/BlockStructureService.fs

```diff
@@ -97,6 +97,8 @@ type internal FSharpBlockStructureService(checker: FSharpChecker, projectInfoMan
                 | Some parsedInput ->
                     let blockSpans =
                         Structure.getOutliningRanges (sourceText.Lines |> Seq.map (fun x -> x.ToString()) |> Seq.toArray) parsedInput
+                        // it's crucial to not return more than one BlockSpan, othewise Roslyn crashes Visual Studio when user hover 
```

Comment:
Comment seems unrelated to this PR - is it?  I have no objection to adding this, just curious.